{"name":"Verify WithDraw in Transaction","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"expected [true] but found [false]","trace":"java.lang.AssertionError: expected [true] but found [false]\n\tat org.testng.Assert.fail(Assert.java:96)\n\tat org.testng.Assert.failNotEquals(Assert.java:776)\n\tat org.testng.Assert.assertTrue(Assert.java:44)\n\tat org.testng.Assert.assertTrue(Assert.java:54)\n\tat tests.WithDrawTest.WithDrawInCorrect(WithDrawTest.java:49)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\n"},"stage":"finished","description":"Testcase Description : Verify WithDraw in Transactions","steps":[{"name":"login with username: truong922946@gmail.com and password : Truong922946 step...","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"strUsername","value":"truong922946@gmail.com"},{"name":"strPassword","value":"Truong922946"}],"start":1605694534726,"stop":1605694545655}],"attachments":[{"name":"Page screenshot","source":"606bb124-e81d-4db0-a05c-bf8e6b391137-attachment","type":"image/png"},{"name":"WithDrawInCorrect failed and screenshot taken!","source":"1b40347b-5343-40be-82e6-8995184ddbb8-attachment","type":"text/plain"}],"parameters":[],"start":1605694534725,"stop":1605694570867,"uuid":"2a50d79f-fa04-44d0-8c07-b1800fd5eafd","historyId":"fb983bb36e938e312b712cae1c39fbbe","fullName":"tests.WithDrawTest.WithDrawInCorrect","labels":[{"name":"package","value":"tests.WithDrawTest"},{"name":"testClass","value":"tests.WithDrawTest"},{"name":"testMethod","value":"WithDrawInCorrect"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Selenium-framwork"},{"name":"subSuite","value":"tests.WithDrawTest"},{"name":"host","value":"BAEMINs-MacBook-Pro.local"},{"name":"thread","value":"12576@BAEMINs-MacBook-Pro.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"severity","value":"normal"}],"links":[]}